index.bs

```diff
@@ -1824,11 +1827,25 @@ a numbered step. If outdented, it (today) is rendered as a bullet in the midst o
 
                 :   is present
                 ::  If <code>|pkOptions|.{{PublicKeyCredentialCreationOptions/rp}}.{{PublicKeyCredentialRpEntity/id}}</code> [=is not a
-                    registrable domain suffix of and is not equal to=] |effectiveDomain|, throw a "{{SecurityError}}" {{DOMException}}.
+                    registrable domain suffix of and is not equal to=] |effectiveDomain|, and if the client
+                
+                    <dl class="switch">
+                        :   supports [[#sctn-related-origins|related origin requests]]
+                        ::  1. Let |rpIdRequested| be the value of <code>|pkOptions|.{{PublicKeyCredentialCreationOptions/rp}}.{{PublicKeyCredentialRpEntity/id}}</code>.
 
-                :   Is not present
-                ::  Set <code>|pkOptions|.{{PublicKeyCredentialCreationOptions/rp}}.{{PublicKeyCredentialRpEntity/id}}</code> to
-                    |effectiveDomain|.
+                            1. If the result of running the [[#sctn-validating-relation-origin|related origins validation procedure]] is
+                                <dl class="switch">
+                                    :   `false`
+                                    ::  throw a "{{SecurityError}}" {{DOMException}}.
+                                    :   `true`
+                                    ::  [=break=].
+                                </dl>
```

Comment:
I don't think there's any loop to `[=break=]` here anymore. Looks like this now reduces to just "If the result of running the related origins validation procedure is [FALSE], throw a "SecurityError" DOMException."?